SQL_module/SubjectForm_Task2/db_connection.php

```diff
@@ -0,0 +1,12 @@
+<?php
+function db_connection(){
```

Comment:
Please use consistent naming convention of functions and stick to the specific naming. Either _underscore_ or camelCase or PascalCase. Just please keep them consistent.